pkgs/applications/science/math/readstat/default.nix

```diff
@@ -16,8 +16,22 @@ stdenv.mkDerivation rec {
       url = "https://github.com/WizardMac/ReadStat/commit/211c342a1cfe46fb7fb984730dd7a29ff4752f35.patch";
       hash = "sha256-nkaEgusylVu7NtzSzBklBuOnqO9qJPovf0qn9tTE6ls=";
     })
+
+    # Backport use-after-free:
+    #   https://github.com/WizardMac/ReadStat/pull/298
+    (fetchpatch {
+      url = "https://github.com/WizardMac/ReadStat/commit/718d49155e327471ed9bf4a8c157f849f285b46c.patch";
+      hash = "sha256-9hmuFa05b4JlxSzquIxXArOGhbi27A+3y5gH1IDg+R0=";
+    })
   ];
 
+  postPatch = ''
+    # Drop blanket -Werror to fix builds fore newer toolchains, like
+    # gcc-13.
+    substituteInPlace Makefile.am \
+      --replace ' -Werror ' ' '
+  '';
+
```

Comment:
```suggestion
```
this is no longer needed.

Reply:
OK, I give up. Removed.

Reply:
yeah-- i flip flop on `-Werror`. cases like this where there is a use after free seems like it did good, many others are just a waste of time.